app/controllers/admin/configs_controller.rb

```diff
@@ -115,8 +115,7 @@ class ConfigsController < Admin::ApplicationController
         facebook_key
         facebook_secret
         invite_only_mode
-        allow_email_password_registration
-        allow_email_password_login
+        allow_both_email_signup_and_login
```

Comment:
Seeing as we are no longer passing the `allow_email_password_registration` and `allow_email_password_login` through params, I deleted them from ALLOWED_PARAMS